src/Tests/Microsoft.NET.Build.Tests/GiventThatWeWantDesignerSupport.cs

```diff
@@ -14,12 +14,18 @@ public GivenThatWeWantDesignerSupport(ITestOutputHelper log) : base(log)
         }
 
         [Theory]
-        [InlineData("net46")]
-        [InlineData("netcoreapp3.0")]
-        [InlineData("net5.0-windows")]
-        public void It_provides_runtime_configuration_and_shadow_copy_files_via_outputgroup(string targetFramework)
+        [InlineData("net46", "true")]
+        [InlineData("net46", "false")]
```

Comment:
Self-contained is something that only applies to .NET Core, so we can remove a case here.

```suggestion
        [InlineData("net46", "false")]
```